### PR TITLE
E2E: Goal /create

### LINKED ISSUE #1642: E2E: Goal /create
- ways to open dialog: menu/hotkey
- clean fields at first time 
- title and project is required, form can't be submitted
- title can be located on goal page
- description is empty by default
  - placeholder in empty case
  - description in filled case
- project
  - empty by default
  - current project from project page
  - most used project in case of some created goals
- priority is Medium by default, change is available and can be located on goal page
- owner is current user by default, change is available and can be located on goal page
- state is Draft by default, change is available and can be located on goal page
- estimate is empty by default, change is available and can be located on goal page (any kind: year, quarter, exact date)
- ways to create
  - create only
  - create and go
  - create one more

### PR DESCRIPTION
**Pull request by LamaEats**
Add specs for goal create functionality
Also add server tasks for db manipulation

## PR includes

- [x] Specs

## Related issues

Resolve #1642

### CODE REVIEW

File: src/components/CommonCombobox.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { forwardRef } from 'react';
+import { ComboBox } from '@taskany/bricks';
+
+import { combobox } from '../utils/domObjects';
+
+export const CommonCombobox = forwardRef<HTMLDivElement, React.ComponentProps<typeof ComboBox>>((props, ref) => {
```
Comment (awinogradov) [inline src/components/CommonCombobox.tsx:6]: Add this link to code plz https://github.com/taskany-inc/bricks/issues/506

File: src/components/CommonDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { forwardRef } from 'react';
+import { Dropdown } from '@taskany/bricks';
+
+import { combobox } from '../utils/domObjects';
+
+export const CommonDropdown = forwardRef<HTMLDivElement, React.ComponentProps<typeof Dropdown>>((props, ref) => {
```
Comment (awinogradov) [inline src/components/CommonDropdown.tsx:6]: Add this link to code plz https://github.com/taskany-inc/bricks/issues/506

**Review by awinogradov — APPROVED** [review]

File: cypress/e2e/goals.cy.ts [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        title: testProjectForGoals,
+        key: projectKey,
+        ownerEmail: Cypress.env('ADMIN_EMAIL'),
+    }).then((p: any) => {
+        Cypress.env('stubProject', p.id);
+    });
+});
+
+after(() => {
+    cy.log('env', Cypress.env());
```
Comment (9teen90nine) [inline cypress/e2e/goals.cy.ts:57]: Is it necessary?

**Review by 9teen90nine — APPROVED** [review]

File: cypress/e2e/goals.cy.ts (hunk 3 above)
Comment (LamaEats) [reply cypress/e2e/goals.cy.ts:57]: nope, will remove it

**Pull request merged**